MediaBrowser.Providers/Manager/GenericPriorityQueue.cs

```diff
@@ -3,7 +3,7 @@
 using System.Collections.Generic;
 using System.Runtime.CompilerServices;
 
-namespace Priority_Queue
```

Comment:
Leave this one, we should try and replace it.

Reply:
I side here with @Bond-009 - this just highlights it being an embedded 3rd party we want eventually to extract.
Maybe add a `FIXME` or `TODO` here and leave the namespace be like it was?